apps/openmw/mwworld/actiontake.cpp

```diff
@@ -14,6 +14,9 @@ namespace MWWorld
 
     void ActionTake::executeImp (const Ptr& actor)
     {
+        //No need to do anything if moving items from the player's inventory back into player's inventory
+        if (actor == MWBase::Environment::get().getWorld()->getPlayerPtr() && getTarget().getCell() == 0)
```

Comment:
I have no idea why this check was added, and it doesn't look correct to me either.